src/Mount.hpp

```diff
@@ -28,6 +28,13 @@ class TMC2209Stepper;
 #define TARGET_STRING      B01000
 #define CURRENT_STRING     B10000
 
+enum HomingState {
+  HOMING_PIN_IDLE = -1,
+  HOMING_PIN_FINDING_START = 0,
+  HOMING_PIN_FINDING_END = 1,
+  HOMING_PIN_FOUND 
```

Comment:
if we already define explicit values for some of the enums, we should make it consistend and also define one for HOMING_PIN_FOUND. Also there is no reason to start here with -1. In case one would want to use these enum values in boolean expressions, HOMING_PIN_FINDING_START would be the one evaluating to `false` ... i would rather say that IDLE is the one matching `false`.

In general i would not recommend using explicit values here because the enum values should be used by name instead of raw values.